### ISSUE TITLE
Icon Only Ghost Button and Radio selection dots are invisible in Windows High Contrast

### ISSUE THREAD

**Issue by Paul-Grogan** [carbon-design-system/carbon#6748]
## What package(s) are you using?

- [x] `carbon-components`
- [ ] `carbon-components-react`

## Detailed description

> Describe in detail the issue you're having.

With Windows10's High Contrast mode enabled, using the Edge (Chromium) browser, the Icon in the Icon only Ghost button is not visible. Similarly, the dot in a Radio button is not visible either.
This can be observed at the sample: https://www.carbondesignsystem.com/components/button

> Is this issue related to a specific component?

Button and Radio Button 

> What did you expect to happen? What happened instead? What would you like to
> see changed?

Expected to see the icon in the Icon Only button, and the dot in the selected radio button

> What browser are you working in?

Identified on the Edge (Chromium) browser, version 85.0.564.41 (Official build) (64-bit)

> What version of the Carbon Design System are you using?

Going to assume the latest because I reproduced it on the Sample ages.

> What offering/product do you work on? Any pressing ship or release dates we
> should be aware of?

WFfH Application Framework, expecting to complete our first ACS certification by the end of September. Which this will impact.

## Steps to reproduce the issue

1. Step one
Enable Windows 10's High Contrast mode

2. Step two
Launch (Edge Chromium) and navigate to the Live Demo section of the Button sample:, https://www.carbondesignsystem.com/components/button/usage/#live-demo

3. Step three
From the type menu change Button to Icon Only.
Set the Kind to Ghost.

This will demonstrate both the invisible icon and the radio button's dot.

## Additional information

- Screenshots ![000656](https://user-images.githubusercontent.com/70535014/91746564-b6b24b00-ebb4-11ea-8233-8344bfc0a60d.png)

**Comment by Paul-Grogan** [comment]
Just noticed that the Radio Selected Dot is already reported in https://github.com/carbon-design-system/carbon/issues/2485

**Comment by wkeese** [comment]
This is still broken, at least on Firefox.  See https://www.carbondesignsystem.com/components/button/usage/.

<img width="969" alt="Screen Shot 2021-01-14 at 8 50 30" src="https://user-images.githubusercontent.com/69599/104526414-abccf400-5645-11eb-8f8e-ee51f6963a95.png">

**Comment by tw15egan** [comment]
I'm unable to reproduce this as well, here is the website running Firefox:

<img width="886" alt="Screen Shot 2021-01-21 at 11 31 38 AM" src="https://user-images.githubusercontent.com/11928039/105402411-3c727980-5bdc-11eb-88d7-1db3788a81ab.png">

Storybook version running Firefox:
<img width="101" alt="Screen Shot 2021-01-21 at 11 30 04 AM" src="https://user-images.githubusercontent.com/11928039/105402483-5449fd80-5bdc-11eb-9dbe-2a9d2dbcccb6.png">

Storybook running Edge
<img width="134" alt="Screen Shot 2021-01-21 at 11 30 24 AM" src="https://user-images.githubusercontent.com/11928039/105402519-60ce5600-5bdc-11eb-8780-f440efcc57d6.png">

**Comment by wkeese** [comment]
TJ - Your screenshots are for PRIMARY buttons.  This ticket is about GHOST buttons.

Having said that...

As for Edge, the original target of this ticket, I can't test the [original site](https://www.carbondesignsystem.com/components/button/usage/) anymore because the "Variant Selector" (and also "Theme selector") dropdowns are currently broken (on Edge).  The storybook site is working.

On Firefox, the original page is working, but on storybook, the "label + icon" case is working but the "icon only" case is not:

<img width="843" alt="Screen Shot 2021-01-22 at 9 47 45" src="https://user-images.githubusercontent.com/69599/105430633-4b6a3200-5c97-11eb-86f4-9738e6aeeb17.png">

(Note that this is the PLAYGROUND page as that's the only way to get an icon-only ghost button.)

Icon only Ghost buttons do work after toggling on the experimental `prefers-contrast` switch from `about:config`... but since "label + icon" ghost button icons work without that flag, why not do the same thing for icon-only ghost buttons?

Note: This is all confusing since ghost buttons are getting borders in high contrast, which makes them seem like primary buttons.

**Comment by tw15egan** [comment]
@wkeese Not sure what's going on on your end. I'm able to navigate the original site (and use the variant selector) and choose the Icon-only ghost button:

![image](https://user-images.githubusercontent.com/11928039/105528310-adc43200-5c99-11eb-99cf-d58851bfd8b4.png)

![image](https://user-images.githubusercontent.com/11928039/105528388-cd5b5a80-5c99-11eb-9fc4-a89198d7fff8.png)

(Firefox)
![image](https://user-images.githubusercontent.com/11928039/105528482-e2d08480-5c99-11eb-92b0-282f4b415897.png)

**Comment by tw15egan** [comment]
The only thing I can think of is to ensure you're running the latest version of edge, and you've enabled High Contrast mode at the system level

![image](https://user-images.githubusercontent.com/11928039/105528717-2cb96a80-5c9a-11eb-919d-6352e3b6bf4b.png)

**Comment by wkeese** [comment]
Oh sorry, I forgot that I was on Edge Legacy instead of Chromium-based Edge.  My manager had a fantasy about supporting both of them.  Unsurprisingly, Chromium Edge is working about the same as Chrome.

So to revise my earlier statements:

1. On Firefox [vanilla](https://react.carbondesignsystem.com/?path=/story/button--playground) and [react](https://www.carbondesignsystem.com/components/button/usage/), for icon-only ghost buttons, this icon is invisible unless you turn on the `prefers-contrast` experimental flag from `about:config`, which triggers the CSS setting of `fill: ButtonText`.  Confusingly, for primary buttons etc, the icon is visible without that flag, because you set `fill: currentColor`.
2. Ghosts buttons in high contrast have a border.  They shouldn't.
3. On IE, on https://www.carbondesignsystem.com/components/button/usage/ live demo, can't adjust selectors, and the radio buttons aren't even displayed.  React icon-only ghost button seems to be working though.
